tools/ci_build/github/linux/ort_minimal/build_minimal_ort_android_baseline_and_report_bin_size.sh

```diff
@@ -29,7 +29,7 @@ python3 /onnxruntime_src/tools/ci_build/build.py \
     --include_ops_by_config /home/onnxruntimedev/.test_data/include_no_operators.config
 
 # set current size limit to BINARY_SIZE_LIMIT_IN_BYTES.
-BINARY_SIZE_LIMIT_IN_BYTES=1256000
+BINARY_SIZE_LIMIT_IN_BYTES=1296000
 echo "The current preset binary size limit is $BINARY_SIZE_LIMIT_IN_BYTES"
```

Comment:
Probably going to need to make optional support able to be excluded if it increases the binary size by ~40K.

Reply:
Will do